src/main/java/com/solvd/webtest/pages/ios/LoginPageIos.java

```diff
@@ -0,0 +1,48 @@
+package com.solvd.webtest.pages.ios;
+
+import com.solvd.webtest.pages.base.LoginPageBase;
+import com.zebrunner.carina.utils.factory.DeviceType;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.time.Duration;
+
+@DeviceType(pageType = DeviceType.Type.IOS_PHONE, parentClass = LoginPageBase.class)
+public class LoginPageIos extends LoginPageBase {
+    @FindBy(id = "userid")
+    private ExtendedWebElement usernameField;
+
+    @FindBy(id = "signin-continue-btn")
+    private ExtendedWebElement continueButton;
+
+    @FindBy(id = "signin-error-msg")
+    private ExtendedWebElement signinError;
+
+    @FindBy(id = "user-info")
+    private ExtendedWebElement userInfo;
+
+    public LoginPageIos(WebDriver driver) {
+        super(driver);
+        setUiLoadedMarker(usernameField);
+    }
+
+    @Override
+    public boolean isOpen() {
+        super.isOpen();
+        return getDriver().getCurrentUrl().startsWith(url);
+    }
+
+    public boolean tryLogin(String username) {
```

Comment:
As I pointed out previously, I am not big fan of calling methods try. It should be just Login. However in this case it should be called isLogged since you return boolean value. Look at naming convention.

Reply:
It sounds good to me 😅 I would say `isLogged` sounds like just checking some condition and `tryLogin` signals some action besides getting bool value.